src/test/java/gov/usgs/wma/statistics/model/JsonDataBuilderTest.java

```diff
@@ -353,6 +358,33 @@ public void test_intermediateValues() {
 
 		assertEquals(individuals, collection);
 	}
+	
+	@Test
+	public void test_addingIntermediateValues_uniquely() {
+		// we had a bug where the current month was added twice because of the seemingly required order of operations
+		List<Value> values6a = new ArrayList<>();
+		List<Value> values6b = new ArrayList<>();
+		List<Value> values7  = new ArrayList<>();
+		
+		values6a.add(SAMPLE_1);
+		values6a.add(SAMPLE_2);
+		values6b.addAll(values6a);
+		values7.add(SAMPLE_3);
+		values7.add(SAMPLE_4);
+		
+		assertEquals("While ensuring an emptly list to start, the size was not zero",
+				0,data.intermediateValuesList.size());
+		data.intermediateValues(values6a);
+		assertEquals("Upon added a couple new month values, the size should increase",
+				2,data.intermediateValuesList.size());
+		data.intermediateValues(values7);
+		assertEquals("Upon added a couple new month values, the size should increase",
+				4,data.intermediateValuesList.size());
+		data.intermediateValues(values6b);
+		assertEquals("Upon added a couple duplicate month values, the size should remain the same "
+				+ "because those values from a duplicated month should not be added multiple times",
+				4,data.intermediateValuesList.size());
+	}
```

Comment:
Here is the test I wrote that first failed when the new code was not added and succeeded once I added the new code.

Reply:
Nice! Instead of `Upon added a` can we put in `Upon adding a`? in the assertion text?

Reply:
changed tense